tests/plugin/interface_test.py

```diff
@@ -0,0 +1,9 @@
+from mythril.plugin.interface import MythrilCLIPlugin, MythrilPlugin
+
+
+def test_construct_cli_plugin():
+    _ = MythrilCLIPlugin()
+
+
+def test_construct_mythril_plugin():
+    _ = MythrilPlugin
```

Comment:
shouldn't an instance be created here? In this case no difference anyway, but if the class is extended in the future it might not get tested.

Reply:
I'm going to make this class abstract, so it won't be constructable anyway. This test is not  very useful 🙃